src/annotator/integrations/test/html-test.js

```diff
@@ -178,6 +178,87 @@ describe('HTMLIntegration', () => {
         integration.fitSideBySide({ expanded: false });
         assert.deepEqual(getMargins(), [null, null]);
       });
+
+      context('main content area has margin:auto', () => {
+        function getComputedMargins(element) {
+          const leftMargin = Math.floor(
+            parseInt(window.getComputedStyle(element).marginLeft, 10)
+          );
+          const rightMargin = Math.floor(
+            parseInt(window.getComputedStyle(element).marginRight, 10)
+          );
+          return [leftMargin, rightMargin];
+        }
+
+        // Add a style node to set a max-width and auto margins on the body
+        function appendBodyStyles(document_) {
+          const el = document_.createElement('style');
+          el.type = 'text/css';
+          el.textContent = 'body { margin: 0 auto; max-width: 400px }';
+          el.classList.add('js-style-test');
+          document_.body.appendChild(el);
+        }
+
+        beforeEach(() => {
+          appendBodyStyles(document);
+          // In these tests, we're treating the body element as the
+          // main content area.
+          //
+          // `guessMainContent` area is called _after_ a right margin is set
+          // on the body, so we'll return here the updated computed left and
+          // right position of the body to emulate a real-life result
+          fakeGuessMainContentArea.callsFake(bodyEl => {
+            const margins = getComputedMargins(bodyEl);
+            return { left: margins[0], right: window.innerWidth - margins[1] };
+          });
+        });
+
+        afterEach(() => {
+          // Remove test styles
+          const elements = document.querySelectorAll('.js-style-test');
```

Comment:
The `NodeList` returned by `querySelectorAll` has a `forEach` so you can also do `document.querySelectorAll(...).forEach(el => el.remove())`.